### PR TITLE
Add e2e tests for minikube's GPU support. 

### PR DESCRIPTION
**Pull request by mindprince**
Here's how I ran these locally:

    # Go to the directory with minikube src
    cd ~/go/src/k8s.io/minikube

    # Build the test binary
    make e2e-linux-amd64

    # Go the test/integration directory.
    # This was needed to get testdata working.
    cd test/integration

    # Run the tests.
    ../../out/e2e-linux-amd64 \
        -minikube-start-args="--vm-driver=kvm2 --gpu" \
        -minikube-args="--v=10 --logtostderr" \
        -test.v \
        -test.timeout=30m \
        -test.run .*Gpu.* \
        -binary=../../out/minikube-linux-amd64

@balopat @dlorenc @vishh @aaron-prindle

### CODE REVIEW

**Comment by balopat** [comment]
@minikube-bot ok to test

**Comment by zmhassan** [comment]
Where can I find documentation on utilizing this feature minikube with:
- nvidia-driver-installer: enabled
- nvidia-gpu-device-plugin: enabled

@mindprince thanks for contributing this feature

**Comment by tstromberg** [comment]
Any chance to make the changes which @balopat asked for?

**Comment by tstromberg** [comment]
Any hope for resurrecting this PR from the merge conflicts? I feel like it's doing the right thing and would love to see it in minikube. Thanks for all the hard work you've put into it so far.

**Comment by mindprince** [comment]
I haven't been able to get cycles to do this. I will try sometime this week
(or weekend), otherwise I will ask someone else to take over this.

--
Rohit

>

File: test/integration/gpu_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
+// +build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```
Comment (RA489) [inline test/integration/gpu_test.go]: s/2016/2019

**Comment by tstromberg** [comment]
Thank you for the contribution. Due to recent inactivity, I am marking this PR as closed until the issues can be resolved. The issue of GPU testing remains open, and something we would like to address after the 1.0 release.

Please re-open once this PR has been updated. Thank you again!

**Pull request closed without merging**